stay-in-tarkovconfig.json

```diff
@@ -60,7 +60,7 @@
 	{
 		"DisplayName": "SIT Version",
         "Category": "SIT Server Settings",
-		"Description": "Sets the SIT version to install. THE MAJOR VERSION MUST MATCH THE SPT VERSION FOR SPT AKI",
+		"Description": "Sets the SIT version to install. THE MAJOR VERSION MUST MATCH THE SPT VERSION FOR SPT AKI. Example: When 1.6.4 with AKI 3.8.0-2dd4d9 is selected, you must have 3.8.0 as the SPT AKI version.",
```

Comment:
The example suggests 3.8.1 can never been used, because there is only one SIT version included? So what's the point of having 3.8.1?

Reply:
SIT for 3.8.1 is supposed to be released soon. You are correct though. The example does suggest that.